src/menu/menuhelpers.rs

```diff
@@ -18,21 +18,31 @@ pub const SUB_TEXT_SCALE_DOWN: f32 = 25.0;
 
 pub static GAME_MODE_UNEXPECTEDLY_NONE: &str = "[!] GameMode unexpectedly None";
 
+#[repr(u8)]
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum MenuState {
+    ChooseMode,
+    Start,
+    Settings,
+    InputConfig,
+}
+
+#[repr(u8)]
```

Comment:
I don't think you need the repr(u8) on any of these enums?

Reply:
Sounds like a godbolt adventure. Or my amazing and not much tested https://github.com/wcampbell0x2a/cargo-rz-ghidra

Reply:
pretty cool tho, it uses `mov byte, {n}` instead of `mov dword, {n}`. I don't know if it gives any performance boosts but eh. Cool.